lib/launchers/base.js

```diff
@@ -24,7 +24,9 @@ var BaseLauncher = function (id, emitter) {
   Object.keys(EventEmitter.prototype).forEach(function (method) {
     this[method] = EventEmitter.prototype[method]
   }, this)
-  KarmaEventEmitter.call(this)
+
+  this.bind = KarmaEventEmitter.prototype.bind.bind(this)
```

Comment:
This trick was needed because of `cannot calls KarmaEventEmitter without new operator` like error. I have plan to do further refactoring, and transform BaseLauncher into regular ES6 class.